admin/used_flusher.go

```diff
@@ -47,6 +53,27 @@ func (u *usedFlusher) Deployment(id string) {
 	u.deployments[id] = true
 }
 
+func (u *usedFlusher) User(id string) {
+	u.mu.Lock()
+	defer u.mu.Unlock()
+
+	u.users[id] = true
+}
+
+func (u *usedFlusher) UserTokens(id string) {
+	u.mu.Lock()
+	defer u.mu.Unlock()
+
+	u.userTokens[id] = true
+}
+
+func (u *usedFlusher) ServiceTokens(id string) {
+	u.mu.Lock()
+	defer u.mu.Unlock()
+
+	u.serviceTokens[id] = true
+}
```

Comment:
Make the function names singular like `Deployment` and `User` (i.e. `UserToken` instead of `UserTokens`)